test/table-test.js

```diff
@@ -542,3 +542,88 @@ describe("__table", () => {
     );
   });
 });
+
+describe("getTypeValidator filters accurately", () => {
+  let source;
+  beforeEach(() => {
+    source = [
+      {label: "string", value: "string"},
+      {label: "object", value: {}},
+      {label: "buffer", value: new ArrayBuffer()},
+      {label: "boolean", value: true},
+      {label: "array", value: [1, 2, 3]},
+      {label: "number", value: 10},
+      {label: "date", value: new Date(1)},
+      // eslint-disable-next-line no-undef
+      {label: "bigint", value: BigInt(10)},
+      {label: "null", value: null},
+      {label: "NaN", value: NaN},
+      {label: "undefined"}
+    ];
+  });
```

Comment:
i don't think you need to redefine `source` before every test case, since `.filter` creates a copy of the array instead of filtering in place.

```suggestion
  let source = [
    {label: "string", value: "string"},
    {label: "object", value: {}},
    {label: "buffer", value: new ArrayBuffer()},
    {label: "boolean", value: true},
    {label: "array", value: [1, 2, 3]},
    {label: "number", value: 10},
    {label: "date", value: new Date(1)},
    // eslint-disable-next-line no-undef
    {label: "bigint", value: BigInt(10)},
    {label: "null", value: null},
    {label: "NaN", value: NaN},
    {label: "undefined"}
   ];
```

Reply:
oh also why do you need the `// eslint-disable-next-line no-undef`?

Reply:
Thanks! Because this version of eslint doesn't recognize BigInt 🤷